bulk-model-sync-mps/src/main/kotlin/org/modelix/mps/model/sync/bulk/CompositeFilter.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.mps.model.sync.bulk
+
+import org.modelix.model.api.INode
+import org.modelix.model.sync.bulk.ModelSynchronizer
+
+class CompositeFilter(private val filters: Collection<ModelSynchronizer.IFilter>) : ModelSynchronizer.IFilter {
```

Comment:
I kept the unordered collection type so users can use unordered collections if they do not care about the order.
I added a comment about evaluation order, in case the collection is ordered.